hazelcast/src/main/java/com/hazelcast/internal/config/AbstractConfigLocator.java

```diff
@@ -165,6 +165,21 @@ protected boolean loadConfigurationFromClasspath(String configFileName) {
         }
     }
 
+    protected boolean loadConfigurationFromClasspath(String configFilePrefix, Collection<String> acceptedSuffixes) {
```

Comment:
Minor: the order in which accepted suffixes are tried is determined outside of this method by the actual type of the `acceptedSuffixes` `Collection`. If that changes in the future for some reason, the tests that depend on the precedence of `.yaml` over `.yml` may break etc. But this is really minor since this is all private API and we have the same 'issue' in other places in the config code base, too.

Reply:
Added `YamlConfigBuilderTest.testYamlSuffixSearchSequence`